saleor/graphql/checkout/schema.py

```diff
@@ -48,14 +48,14 @@ class CheckoutQueries(graphene.ObjectType):
     def resolve_checkout(self, *_args, token):
         return resolve_checkout(token)
 
-    @permission_required(OrderPermissions.MANAGE_ORDERS)
+    @permission_required(CheckoutPermissions.MANAGE_CHECKOUTS)
     def resolve_checkouts(self, *_args, **_kwargs):
         resolve_checkouts()
```

Comment:
It won't work. This method doesn't return anything

Reply:
Ah yeah good point!

Reply:
Your permission was ok. I was meaning that method `resolve_checkouts` doesnt return anything - missing `return` in line with `resolve_checkouts()`. You could just add it :)